0033-relationship-between-protocols/0033-relationship-between-protocols.md

```diff
@@ -84,10 +92,12 @@ This is an enlarged diagram of the above diagram, that shows mainly `Node A`, `N
     - DCP is a protocol built on top of ILP, and is used in order to exchange node information such as an ILP address.
     - DCP data is put into the extensible data area of ILP packets.
   - (8) RBP
-    - RBP is a protocol built on top of ILP, and is used for transferring routing information to build routing tables.
-    - RBP data is put into the extensible data area of ILP packets.
+    - A Route Broadcasting Protocol is thought to be a protocol built on top of ILP, and would be used for transferring routing information to build routing tables.
+    - RBP data would be put into the extensible data area of ILP packets.
+    - RBP is an idea that does not exist yet.
```

Comment:
I made this section hypothetical. If we want to completely remove it, we need to change the svgs and the numbering.

Reply:
To me, the third point is already implied by the first one

Reply:
We can imply that something does exist, but it is not part of the official spec but simply EXPERIMENTAL, which we can still link to.